### PR TITLE
Feat/#25 connect preprocessing online

### PR DESCRIPTION
**Pull request by sheagrief**

### CODE REVIEW

**Review by taskooh — APPROVED** [review]
LGTM!

File: src/she/plaintext.rs [hunk 1]
```diff
@@ -28,6 +28,16 @@ impl Plaintexts {
         Plaintexts { vals: res }
     }
 
+    pub fn restricted_rand<T: Rng>(params: &SHEParameters, rng: &mut T) -> Plaintexts {
+        let upper_bound = 1000000000;
+        let lower_bound = 100000;
```
Comment (taskooh) [inline src/she/plaintext.rs:33]: How did you decided this upper bound & lower bound?

Comment (sheagrief) [reply src/she/plaintext.rs:33]: need to satisfy the following approximate conditions.
Lower Bound > maximum value of possible secret input (approximately 10,000 in this case) * number of participants
Upper Bound * number of participants < period of the ScalarField of edwards_bls12_377 (≒10^75)

Comment (taskooh) [reply src/she/plaintext.rs:37]: Got it! I added comment

**Pull request merged**